frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/MainActivity.java

```diff
@@ -352,13 +353,8 @@ public boolean onShowFileChooser(WebView webView, ValueCallback<Uri[]> filePathC
                 MainActivity.this.filePathCallback = filePathCallback;
                 String[] mimeTypes = fileChooserParams.getAcceptTypes();
                 String fileType = "*/*";
-                if (mimeTypes.length == 1) {
-                    // import notes form uses .txt file type, but is not supported here.
-                    if (".txt".equals(mimeTypes[0])) {
-                        fileType = "text/plain";
-                    } else if (MimeTypeMap.getSingleton().hasMimeType(mimeTypes[0])) {
-                        fileType = mimeTypes[0];
-                    }
```

Comment:
Checking this again and seeing that the previous commit to restrict to the right file types (https://github.com/BitBoxSwiss/bitbox-wallet-app/commit/045aa86aa767f034c38b522e0dc1308dbc1e3540) is undone.

You write

>  The (android) filer picker shows .jsonl files as binary (i.e. it doesn't recognize it),

I guess it's not as easy as adding a ".jsonl" check same as the ".txt" check above? :thinking:

Reply:
Yes, checking for both `jsonl` and `txt` exclusively like before does not work, hence the full fallback to `*/*`.  It would probably require a more involved custom change, which I'm not sure how to do.